update readme (#2)
tryber/sd-05-summaries: README.md

# Boas vindas ao repositório de compartilhamento de resumos de cursos da Trybe!

> <strong>O objetivo deste repositório</strong>

Reviewer: ```suggestion
<strong>Objetivo do repositório</strong>
```

> Aqui, é uma ferramenta para colaborar com os estudos uns dos outros, via o compartilhamento de resumos de conteúdos. De fato, na hora de praticar ou querer relembrar o curso, é muito mais fácil consultar uma fonte só do que uma mistura do caderno, favoritos chrome, aulas trybe e slack, certo?

Reviewer: ```suggestion
Esta é uma ferramenta para colaborar com os estudos uns dos outros, via o compartilhamento de resumos de conteúdos. Na hora de praticar ou relembrar os conteúdos, é mais fácil consultar uma fonte única do que uma mistura do seu caderno, favoritos do chrome, aulas da trybe e slack, certo?
```

<em>Aqui você pode:

> <br>Consultar resumos do curso, simplesmente navegando os pull requests dos outros alunos; <br>e/ou

Reviewer: ```suggestion
<br>Consultar resumos de aulas do curso através da navegação pelos <em>Pull Requests</em> das pessoas estudantes; <br>e/ou
```

> Participar adicionando resumos seus, utilizando uma branch específica e um Pull Request para colocar seus arquivos.</em><br>

Reviewer: ```suggestion
Participar adicionando seus resumos, utilizando uma branch específica e um Pull Request para colocar seus arquivos.</em><br>
```

<strong>Consultar os resumos</strong><br><br>

> À medida que você e os outros estudantes forem entregando os resumos, estará avisado. Pode ir consultando, revisando e comentando os resumos dos outros, para dar parabéns e/ou caso for encontrar algum erro a ser corrigido!

Reviewer: ```suggestion
À medida que você e as outras pessoas estudantes forem adicionando os resumos, vocês poderão revisar e comentar para dar parabéns e/ou sugerir alterações.
```

> Tamos juntos!

Reviewer: ```suggestion
Estamos juntos!
```

<strong>Instruções para contribuir</strong><br><br>

> Para contribuir, você deverá seguir as instruções a seguir. Fique atento a cada passo, e se tiver dúvida, mande pelo canal Slack #studyteams!

Reviewer: ```suggestion
Você deverá seguir as instruções a seguir. Fique atenta a cada passo. Caso tenha dúvidas, mande mensagem no canal da sua turma!
```

    1. Tenha, de preferência, seu resumo já pronto.

>     É bem melhor para todos poder consultar um trabalho já bem feito e verificado.

Reviewer: ```suggestion
    É melhor poder consultar um trabalho finalizado e verificado.
```

>     Pode ser um arquivo criado por você ou algum PDF útil achado online,

Reviewer: ```suggestion
    Pode ser um arquivo criado por você ou algum conteúdo livre encontrado online,
```

>     desde que encaixe na estrutura clara dos modulos e blocos do curso da Trybe.

Reviewer: ```suggestion
    desde que encaixe na estrutura clara dos módulos e blocos do curso da Trybe.
```

    correspondendo com os modulos e blocos do curso.
    
    5. Aporte sua contribuição, do add ao push:
            ▪ git add . (adicionando todas as mudanças - que estavam em vermelho - ao stage do Git);
            ▪ git status (devem aparecerer listados os arquivos modificados em verde);
            ▪ git commit -m “” (para criar um commit com os arquivos que estão no stage do Git. Dica: deixa claro neste commit o conteúdo que aportou);
            ▪ git push -u nome-da-branch (para enviar o commit para o repositório remoto na primeira vez que fizer o push de uma nova branch)
            ▪ git push (para enviar o commit para o repositório remoto após o passo anterior).
            
    6. Crie um novo Pull Request (PR)
    • Vá até a página de Pull Requests do repositório no GitHub 
    • Clique no botão verde "New pull request"
    • Clique na caixa de seleção "Compare" e escolha a sua branch com atenção
    • Clique no botão verde "Create pull request"
    • Adicione uma descrição para o Pull Request e clique no botão verde "Create pull request"
    • Não se preocupe em preencher mais nada por enquanto! 
    

>     7. Não faça merge. Inclusive, está desabilitado.

Reviewer: ```suggestion
```

>     VQV!!

Reviewer: ```suggestion
    VQV!!
    <em>Repositório idealizado e criado pela estudante @juliettebeaudet </em>
```
